it_hardware/demo/website_theme_apply.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+
+     <function model="web_editor.assets" name="make_scss_customization" context="{'website_id': ref('website.default_website')}">
+        <value eval="'/website/static/src/scss/options/colors/user_color_palette.scss'"/>
+        <value eval="{'o-color-1': '#0b4f9f','o-color-2': '#101c35','o-color-3': '#E7E8EB','o-color-4': '#FFFFFF','o-color-5': '#083B77', 'menu': 1}"/>
+    </function>
+
+    <function model="ir.module.module" name="_theme_load" context="{'apply_new_theme': True}">
+        <value model="ir.module.module" eval="obj().env['ir.module.module'].search([('name', '=', 'theme_graphene')]).ids"/>
+        <value  model="ir.module.module" eval="obj().env.ref('website.default_website')"/>
```

Comment:
```suggestion
        <value eval="ref('website.default_website')"/>
```

Reply:
My bad It's fine

<value  model="ir.module.module" eval="obj().env.ref('website.default_website')"/>